src/main/java/frc/robot/Match.java

```diff
@@ -20,17 +24,48 @@
 
 public class Match extends TimedRobot
 {
+  private ADXRS450_Gyro gyro;
+  private ADXL362 accel;
+  private LinearFilter xAccelFilter;
+  private double previousXAccel;
+  private double previousVelocity;
+  private double totalDistance;
+  private double posX;
+  private double posY;
```

Comment:
Is there any way we can move this code to the drivetrain?  Since it is only used by that subsystem.  Generally you don't want to have too much logic out in `Match.java`, just the scheduler and commands, along with the robot instantiation.

Reply:
The problem with that is that the accelerometer update needs to be run every cycle. Putting it in drivetrain as the default command won't work because other commands take over from drive all the time. I would need to make another subsystem.

Reply:
Ok, I fixed it by making another subsystem specifically for the sensors.

Reply:
Subsystems should have an inherited method named `periodic()`.  You should be able to override it in drivetrain and update the sensors there:
https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/Subsystem.html#periodic()

Reply:
I didn't know about that, that makes a lot more sense than what I did. Thanks!